ansible/library/show_interface.py

```diff
@@ -145,6 +165,16 @@ def collect_interface_status(self):
                         self.int_status[interface]['vlan'] = old.group(5)
                         self.int_status[interface]['oper_state'] = old.group(6)
                         self.int_status[interface]['admin_state'] = old.group(7)
+                    elif internal and include_internal_intfs:
```

Comment:
Is there a need for checking both? Code is fine either way

Reply:
include_internal_intfs is required to check if the internal ports are up